site/docs/getting-started.md

```diff
@@ -5,30 +5,38 @@ redirect_from: "docs/index.html"
 layout: docs
 ---
 
+## What is Stylable
 
+**Stylable** enables you to build reusable, highly-performant components. Each component exposes a style API that maps its internal parts so you can reuse components across teams without sacrificing stylability.
 
-> "New ideas will come along, but they will extend CSS rather than replace it. I believe that the CSS code we write today will be readable by computers 500 years from now."  
->  
-> [Håkon Wium Lie](https://dev.opera.com/articles/css-twenty-years-hakon/), co-creator of CSS 
+* Scopes styles to components so they don't "leak" and clash with other styles.
+* Enables custom pseudo-classes and pseudo-elements that abstract the internal structure of a component. These can then be styled externally.
+* Uses themes so you can apply different look and feel across your web application.
 
+At build time, the preprocessor converts the Stylable CSS into flat, static, valid vanilla CSS that works cross-browser.
 
+## How do I install
 
-At Wix, we agree. We love CSS. It's simple, declarative syntax that is native in browsers and is easily the fastest way to add styles to web pages and web apps. But when writing CSS that is scoped to individual components, developers have to maintain highly-specific selectors, using elaborate conventions to fake namespacing. Writing and maintanining CSS across large teams and large projects can be tricky.
+1. Install **Stylable** from our GitHub [repository](https://github.com/wix/stylable).
 
-We want to extend CSS so that it is easier to use in a component ecosystem, but without losing the aspects of CSS that make it great: declarative, familiar, static and fast. So we created **Stylable**.
+2. Install the **Stylable** [integrations package](https://github.com/wix/stylable-integration).
 
-**Styable** is a preprocessor that allows you to write style rules in CSS syntax, with some extensions that we believe adhere to the spirit of CSS.
+3. While **Stylable** can work with any type system, the most commonly used is React. Install the latest version.
```

Comment:
React is not a Type System. You can say:

"While **Stylable** can work with any component framework, here at Wix we use it with React."